packages/twenty-front/src/modules/object-record/record-show/hooks/useRecordShowContainerTabs.ts

```diff
@@ -209,20 +221,36 @@ export const useRecordShowContainerTabs = (
         },
       },
     }),
+    [isPermissionsV2Enabled],
+  );
+
+  const baseRecordLayout = BASE_RECORD_LAYOUT;
+
+  const getObjectReadPermission = useRecoilCallback(
```

Comment:
this does not look right to call a method in a reactive component. let's get the whole objectPermissions object here and filter on it below